lightning/checkpoints/checkpoints.go

```diff
@@ -463,7 +465,7 @@ func (cpdb *MySQLCheckpointsDB) Initialize(ctx context.Context, dbInfo map[strin
 		// We do need to capture the error is display a user friendly message
 		// (multiple nodes cannot import the same table) though.
 		stmt, err := tx.PrepareContext(c, fmt.Sprintf(`
-			INSERT INTO %s.%s (task_id, table_name, hash) VALUES (?, ?, ?)
+			INSERT INTO %s.%s (task_id, table_name, hash, table_id) VALUES (?, ?, ?, ?)
```

Comment:
please increase `checkpointTableNameTable` on line 62.

Reply:
updated in https://github.com/pingcap/tidb-lightning/pull/317/commits/bf80f2c769ad59073347981c8cfc71048148ca3a